src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/Introduction.tsx

```diff
@@ -48,8 +52,18 @@ export const VanguardIntroduction: React.SFC<{
   )
 }
 
+const VideoWrapper = styled(Flex)`
+  flex-direction: row;
+  position: relative;
+  padding-bottom: 100px;
+  min-height: 100vh;
+`
+
 const HeaderText = styled(Sans)`
   font-size: 100px;
+  position: absolute;
+  top: 3vw;
+  left: 35vw;
```

Comment:
We probably don't want to set the `left` value hard coded, have you tried making the width 100% and centering?